metric/blockgroup/dUNFOWN.csv

```diff
@@ -1,154 +1,154 @@
-370630018062,1119,1079,1021,1032,1012,1079,1226,1220
-370630018071,493,523,586,533,520,455,404,329
-370630018072,915,794,848,804,820,878,931,994
-370630018073,867,909,776,818,785,755,669,674
-370630018081,650,669,553,597,602,643,643,695
-370630018082,688,789,1017,1070,1255,1414,1881,2104
-370630018091,260,259,265,246,271,308,299,317
-370630018092,724,756,749,638,579,563,508,497
-370630019001,67,68,88,134,147,140,243,372
-370630019002,515,570,683,720,838,936,907,906
-370630020071,598,586,596,482,605,570,668,642
-370630020072,474,471,443,505,389,471,466,492
-370630020073,279,276,332,299,284,237,252,183
-370630020081,242,212,181,175,220,237,214,239
-370630020082,596,667,680,705,669,699,634,642
-370630020091,428,440,389,302,247,249,242,270
-370630020092,116,153,174,234,217,181,208,153
-370630020093,282,327,309,341,340,250,197,195
-370630020131,376,410,390,419,405,432,463,465
-370630020132,177,171,157,174,138,160,147,242
-370630020133,601,624,641,605,609,582,576,558
-370630020151,185,171,149,211,236,238,245,192
-370630020152,170,150,139,150,50,114,108,140
-370630020153,15,15,18,0,11,14,12,14
-370630020161,0,0,0,0,0,0,0,0
-370630020162,161,144,111,97,146,143,117,238
-370630020171,457,473,473,423,508,544,593,647
-370630020172,454,433,412,400,454,393,360,362
-370630020173,501,462,492,506,499,496,452,506
-370630020181,336,344,401,398,371,325,382,473
-370630020182,383,368,424,492,535,565,585,445
-370630020183,665,736,727,642,625,696,802,818
-370630020191,84,51,140,136,116,171,204,154
-370630020192,224,199,236,253,221,264,276,266
-370630020193,546,580,588,496,517,568,537,584
-370630020201,610,588,541,674,697,596,624,574
-370630020202,1115,1212,1214,1311,1278,1415,1326,1555
-370630020211,545,509,506,368,392,294,318,284
-370630020212,252,243,231,260,237,282,289,226
-370630020221,169,162,153,162,129,182,157,133
-370630020222,663,688,599,649,585,580,562,498
-370630020231,169,215,263,236,216,251,198,214
-370630020232,326,270,255,267,293,294,317,250
-370630020241,1192,1174,1117,1072,925,1008,956,1083
-370630020242,288,282,234,231,254,315,366,394
-370630020243,482,478,475,534,405,431,327,331
-370630020251,707,820,944,944,847,717,697,618
-370630020252,765,744,686,668,669,622,568,547
-370630020253,328,395,356,431,370,412,327,303
-370630020261,367,417,428,461,428,424,407,464
-370630020262,542,515,553,467,478,481,372,375
-370630020271,179,188,293,371,378,342,391,506
-370630020272,790,845,778,627,699,495,637,759
-370630020273,0,0,0,0,0,0,15,0
-370630020281,594,658,693,757,770,737,755,773
-370630020282,0,16,0,0,10,14,85,133
-370630020283,155,228,261,362,373,465,535,634
-370630021001,473,431,470,494,478,517,609,605
-370630021002,546,569,604,628,643,652,639,644
-370630021003,452,476,456,464,475,443,442,375
-370630022001,132,146,148,168,151,175,175,182
-370630023001,11,12,23,7,7,13,20,41
-370630023002,0,7,4,1,4,4,0,0
-370639801001,25,39,32,25,0,0,0,0
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
```

Comment:
@BiSeye This one is replacing the dUNFOWN table with the rUNFOWN table data. Be careful to write the dUNFOWN csv from the dUNFOWN table in R. To check it here I compared the first row with the Compass ([https://compass.durhamnc.gov/en/compass/UNFOWN/blockgroup?selected=370630001011]) which shows 37, 48, 43, 34 households in the numerator each recent year.

Reply:
I repushed the correction.  Yes i caught the mistake in my script

Reply:
@BiSeye did you repush to this branch?